.github/workflows/ci.yml

```diff
@@ -0,0 +1,40 @@
+name: CI
+
+on:
+    [push, pull_request]
+
+jobs:
+    test:
+        name: Test on PHP ${{ matrix.php }}
+        runs-on: ubuntu-latest
+
+        strategy:
+            fail-fast: false
+            matrix:
+                php: ['7.1', '7.2', '7.3', '7.4', '8.0']
+
+        steps:
+            -   uses: actions/checkout@v2
+
+            -   name: Setup PHP
+                uses: shivammathur/setup-php@v2
+                with:
+                    php-version: ${{ matrix.php }}
+                    coverage: none
+
+            -   name: Determine composer cache directory
+                id: composer-cache
+                run: echo "::set-output name=directory::$(composer config cache-dir)"
+
+            -   name: Cache composer dependencies
+                uses: actions/cache@v1
+                with:
+                    path: ${{ steps.composer-cache.outputs.directory }}
+                    key: ${{ matrix.php }}-composer-${{ hashFiles('**/composer.lock') }}
+                    restore-keys: ${{ matrix.php }}-composer-
+
+            -   name: Install dependencies
+                run: composer update --no-progress --ansi
+
+            -   name: Run phpunit
+                run: vendor/bin/simple-phpunit
```

Comment:
I suggest adding a separate step doing the `vendor/bin/simple-phpunit install` before that, so that the output of installing phpunit is in a separate step. It makes the logs more readable.

Reply:
Fixed!